spec.html

```diff
@@ -28346,7 +28361,7 @@ <h1>Math.floor ( _x_ )</h1>
 
       <emu-clause id="sec-math.fround">
         <h1>Math.fround ( _x_ )</h1>
-        <p>When `Math.fround` is called with argument _x_, the following steps are taken:</p>
+        <p>When the `Math.fround` method is called with argument _x_, the following steps are taken:</p>
```

Comment:
Every other unary function's summary now says "one argument", so let's keep it consistent.

```suggestion
        <p>When the `Math.fround` method is called with one argument _x_, the following steps are taken:</p>
```